test/pkgs/PackageThatWillBeModified/dyf/ANodeToModify.dyf

```diff
@@ -86,7 +86,7 @@
         {
           "Id": "52c34d69fe42408ca5c857f371168587",
           "Name": "x",
-          "Description": "x value.\n\nvar[]..[]",
+          "Description": "Integer or double value\n\nvar[]..[]",
```

Comment:
@SHKnudsen - I think these changes were likely committed by accident? This seems unrelated to this PR?